contracts/Positions/PositionalMarketManager.sol

```diff
@@ -247,20 +313,21 @@ contract PositionalMarketManager is Initializable, ProxyOwned, ProxyPausable, IP
 
         require(capitalRequirement <= initialMint, "Insufficient capital");
 
-        PositionalMarket market =
-            PositionalMarketFactory(positionalMarketFactory).createMarket(
-                PositionalMarketFactory.PositionCreationMarketParameters(
-                    msg.sender,
-                    sUSD,
-                    priceFeed,
-                    oracleKey,
-                    strikePrice,
-                    [maturity, expiry],
-                    initialMint,
-                    customMarket,
-                    customOracle
-                )
-            );
+        require(_checkMarkets(oracleKey, strikePrice, maturity) == true, "Market already exists");
```

Comment:
Have a different message, e.g. "A market already exists within that timeframe and price buffer"